hyperjumptech/monika
# Feat: reload probes on config change

# Monika PR

## What issue this PR is trying to solve
Monika needs to reload when config file is changed

## How this PR solve the issue
- Added file watcher using `chokidar`

## How to test
- start monika
- while monika is running, change the config file. You should see that monika reloads with new configuration
